app/models/speech.rb

```diff
@@ -1,11 +1,12 @@
 class Speech < ContentItem
   include NewsImage
   include Organisations
+  include People
   include Political
   include Updatable
 
   def contributors
-    (organisations_ordered_by_emphasis + [speaker].flatten + [speaker_without_profile].flatten).compact
+    (organisations_ordered_by_emphasis + people + [speaker].flatten + [speaker_without_profile].flatten).compact.uniq { |contributor| contributor["content_id"] }
```

Comment:
> `{ |contributor| contributor["content_id"] }

What is this block supposed to do?

Reply:
Is `speaker` or `speaker_without_profile` an array? If they are, then they don't need to be made into an array by wrapping them in `[]`. If they aren't arrays, then `flatten` isn't required because the array will only contain one item.